deepchecks/base/display_suite.py

```diff
@@ -54,6 +55,10 @@ def get_display_exists_icon(exists: bool):
 
 def display_suite_result(suite_name: str, results: List[Union[CheckResult, CheckFailure]]):
     """Display results of suite in IPython."""
+    if len(results) == 0:
```

Comment:
I think this should be impossible, maybe we need in a suite creation to ensure there is at least 1 check

Reply:
On second thought it is possible. you can create a suite with only train-test checks, and than run suite with single dataset so nothing will run

Reply:
>I think this should be impossible

it is possible, currently, we do not do any validation for this, not in init method not in run method

>maybe we need in a suite creation to ensure there is at least 1 check

we have the `add` method within the Suite which assumes that the suite is a mutable type and that we can add checks to it after its creation